src/types.ts

```diff
@@ -50,6 +53,7 @@ export interface Marketplace {
   description: string
   logoUrl: string
   bannerUrl: string
+  auctionHouses?: AuctionHouse[]
```

Comment:
This should either be an empty list or filled with auctionHouses we shouldn't make it nullable.